src/bicep/examples/automationAccount/test.ps1

```diff
@@ -0,0 +1 @@
+Write-Output "Successful Runbook Test"
```

Comment:
Let's remove this because it's not being directly referenced. 

Or if you feel strongly that it's a good resource then let's add an example to the README that shows how to use the module to deploy this.